pandas/tests/extension/arrow/test_timestamp.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+from typing import Type
+
+import pytest
+
+import pandas as pd
+from pandas.api.extensions import ExtensionDtype, register_extension_dtype
+
+pytest.importorskip("pyarrow", minversion="0.13.0")
+
+import pyarrow as pa  # isort:skip
+
+from .arrays import ArrowExtensionArray  # isort:skip
+
+
+@register_extension_dtype
+class ArrowTimestampUSDtype(ExtensionDtype):
+
+    type = datetime.datetime
+    kind = "M"
+    name = "arrow_timestamp_us"
+    na_value = pa.NULL
```

Comment:
i thought pyarrow used pa.NA, is that distinct from pa.NULL?

Reply:
They are both the same:

```
In [2]: pyarrow.NULL is pyarrow.NA
Out[2]: True
```

Reply:
`NULL` has been there since ages, `NA` is probably a bit newer.